automl/google/cloud/automl_v1beta1/tables/tables_client.py

```diff
@@ -104,17 +104,18 @@ def __init__(
         else:
             client_info_.user_agent = user_agent
             client_info_.gapic_version = version
```

Comment:
Thanks for the PR!
I think it's good to encourage being explicit with the argument. E.g., prefer argument than using kwargs.
Instead of removing the `client_info` argument, could you change the solution to removing "client_info" from kwargs after this line?

Reply:
Done, thanks!